### PR TITLE
Agent news and features

### PR DESCRIPTION
**Pull request by allelos**
**Designs**:
https://www.figma.com/file/RgebwHKHvTCAlTmIxLsb6l/New-Cloud-and-Agent-navigation-%231629?node-id=535%3A935

**Unseen**:
image

**Seen**:
image

image

### CODE REVIEW

File: src/components/news/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React, { Fragment, useCallback } from "react"
+import { useToggle } from "react-use"
+import { Layer, Flex } from "@netdata/netdata-ui"
+import Header from "./header"
+import Item from "./item"
+import { Container } from "./container"
+
+const News = ({ children, items = [], onCloseClick }) => {
```
Comment (apostolidhs) [inline src/components/news/index.js:8]: ~~onCloseClick~~ -> onClose

**Review by apostolidhs — APPROVED** [review]
Since we store the `last seen` in local storage, could we store the latest news and display the amount of unreadable news in a small badge?

**Review by apostolidhs — COMMENTED** [review]
Also, should we have an empty slate, what do we shaw if there are not news or there was a network error?

File: src/components/header/news/dataSource/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import Prismic from "@prismicio/client"
+
+const apiEndpoint = "https://netdata-news.cdn.prismic.io/api/v2"
+const client = Prismic.client(apiEndpoint)
+
+export const fetchNews = async onSuccess => {
+  const response = await client.query(
+    Prismic.Predicates.at("document.type", "news__announcements"),
+    { pageSize: 100, orderings: "[document.last_publication_date desc]" }
```
[the remark this answers is not in the document]
Comment (allelos) [reply src/components/header/news/dataSource/index.js:9]: Yes this means that the `pageSize` will be 100 entries.  Prismic default is 20 and max is 100. Latest or oldest depends on the `orderings` key. I totally agree with you that 100 is a lot and we should probably do a paginated list that fetches on bottom. But to be honest I don't see even getting to 50 items soon, so we can handle it later!

File: src/components/header/news/index.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import Item from "components/header/item"
+import Pill from "components/header/pill"
+import { fetchNews } from "./dataSource"
+
+const AgentNews = () => {
+  const [value, setValue] = useLocalStorage("news_last_seen")
+  const [news, setNews] = useState([])
+
+  useEffect(() => {
+    fetchNews(({ results }) => setNews(results))
```
[the remark this answers is not in the document]
Comment (allelos) [reply src/components/header/news/index.js:13]: I don't have an objection with this, let's have product tackle it, especially the loading state situation.

**Comment by allelos** [comment]
> Since we store the `last seen` in local storage, could we store the latest news and display the amount of unreadable news in a small badge?

Well, I like counters too but they don't exist in the designs for a reason I guess. Anyways, if we were to go this way we wouldn't have to store the latest news. We could just count how many entries are newer than the last opened date 😃

File: src/components/news/index.js (hunk 1 above)
Comment (allelos) [reply src/components/news/index.js:8]: I insist with `onCloseClick` 😛 . Let's keep it consistent with the documentation helper also!

**Pull request merged**